src/pages/getting-started/troubleshooting.md

```diff
@@ -117,3 +119,65 @@ Non-fatal errors are represented in the _Problem Details_ format, but are embedd
+| `0400-403` | Request ignored due to geographical restrictions | Requests from this country are not allowed. |
+| `0500-401` | Invalid authorization token | The required authorization token is missing or has an invalid format. Add a valid token to the request and try again. |
+| `0501-401` | Invalid user authorization token | The required user token is missing or it has an invalid format. Check the `{headerName}` header value and try again. |
+| `0502-401` | Invalid authorization token signature | The authorization token signature is invalid. Check the authorization token and try again. |
+| `0503-401` | Authorization token expired | The authorization token has expired. Generate a new authorization token and try again. |
+| `0504-401` | Missing required product context | The user authorization token is missing the following required product context: `{productContext}`. Add the required product context to the user account and try again. |
+| `0505-401` | Missing required authorization token scope | The service authorization token is missing the following required scope: `{scope}`. Assign the required scope and try again. |
+| `0506-401` | Missing sandbox write permissions | The user authorization token does not have `DATASET.WRITE` permissions for the `{sandboxName}` sandbox. Check the user permissions and try again. |
+| `0600-429` | Inlet unavailable | The inlet with the ID `{inletId}` is currently unavailable due to a large number of requests. Try again later. |
+| `0601-400` | Inlet inactive | The inlet with the ID `{inletId}` is inactive. Provide an ID of an active inlet and try again. |
+| `0602-400` | Inlet does not exist | The inlet with the ID `{inletId}` does not exist. Provide a valid inlet ID and try again. |
+| `0610-500` | The message could not be published | An internal error occurred during message ingestion. Please try again. If this error persists, contact your Adobe representative. |
+| `0611-400` | Invalid XDM message | The XDM message on the request is invalid. Update your request and try again. |
+| `0612-207` | Message partially ingested | The request contains attributes which are not present in the target schema. These attributes were not published. To publish the missing attributes, add them to your target schema and try again. Otherwise, remove them from future requests. |
+| `4000-400` | Invalid routing set configuration | The routing set with the ID `{routingSetId}` is disabled. Update your routing set configuration and try again. |
+| `4001-400` | Invalid event | The event could not be evaluated due to an error with the XDM content. Update your request and try again. |
+| `4002-404` | Routing set not found | The routing set configuration with the ID `{routingSetId}` was not found. Update the request with a valid routing set ID and try again. |
```

Comment:
Let us also exclude these ones for the moment, and add them into an addendum at a later time.